src/de.hpi.swa.trufflesqueak/src/de/hpi/swa/trufflesqueak/model/AbstractSqueakObjectWithClassAndHash.java

```diff
@@ -183,6 +183,18 @@ public final boolean tryToMark(final boolean currentMarkingFlag) {
         }
     }
 
+    /**
+     * @return <tt>true</tt> if already marked, <tt>false</tt> otherwise
+     */
+    public final boolean tryToUnmark(final boolean currentMarkingFlag) {
```

Comment:
If I see this right, the return value isn't used. Maybe we can just inline this method.